package.json

```diff
@@ -4,34 +4,44 @@
   "author": "Sean McIntyre <smcintyre@xverba.ca>",
   "description": "nom is not npm",
   "contributors": [
-    {"name": "Daniela Baron","email": "dbaron@uncharted.software"},
-    {"name": "Michael Laccetti","email": "michael@laccetti.com"},
-    {"name": "Sean McIntyre","email": "s.mcintyre@xverba.ca"}
+    {
```

Comment:
Didn't intentionally change the formatting, this happened on running `npm install request-promise --save`. The only real change is adding this dependency.